### PR TITLE
Adding a few generalized tags.

### PR DESCRIPTION
**Pull request by IsItBadToMakeAGitHubAccountJustForThis**
In response to #699
Have no idea about how to pick tags and such, so yeah.
ToDo:
- Double check everything
- Add a 'heavy' tag to more serious topics.

### CODE REVIEW

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -810,7 +810,7 @@ label monika_rap:
 
 
 init 5 python:
-    addEvent(Event(eventlabel="monika_horror",category=['drinks'],prompt="Yuri's wine",random=True))
+    addEvent(Event(eventlabel="monika_horror",category=['drinks', 'club'],prompt="Yuri's wine",random=True))
```
Comment (IsItBadToMakeAGitHubAccountJustForThis) [inline Monika After Story/game/script-topics.rpy]: eventlabel is wrong.

**Comment by IsItBadToMakeAGitHubAccountJustForThis** [comment]
About the tags I used:
Dan - topic somehow related to Salvato
Mental disorders - topic mentions depression or anxiety
Psychology - topic contains reflections on human psychology. Including mental disorders but not limited to them (probably missed it in a lot of places)
Social - reflections on society as a whole or more basic "social" concepts as friends or such.
Education - for topics about school and college and education in general
School - topics specifically about school
Growing up - topics about maturing as a person right now fits only into monika_middleschool and probably monika_immortal
Cinema/music/literature/art - self-explanatory
Club - topics mentioning the club (as a place, not as members, they should probably have their own tag) and any.
Romantic - any romantic interactions with Monika like saying "I love you" and such.
Relationship - topics related to player's relationship with Monika (Your future with her, dates, loyalty, such kind of stuff)
Player - topics referring to the player themselves.
MC - topics referring to player as a character
Talent/Creativity - Kinda made those up to add something to monika_creative
Past - topics about Monika's past before the game
Favorites - topics about Monika's likes/dislikes. Should be taken seriously because if done right would help to shape the Monika's voice greatly
Game/game - topics about games and DDLC in particular. Refers to DDLC when written with capital, otherwise games in general
Tropes - I used it to mostly tag anime tropes like tundere and such but can be applied to any tropes in media
Scripting - topics related to Monika changing the game
Challenge - monika_difficulty and monika_completionist seem alike to me so I used this one to unify those
MAS - topics directly talking about this mod
Epiphany - topics about Monika's realization
President - topics mentioning Monika's realization but from a perspective of a club's president role
AI - WOW. SUCH TAG! SO DESCRIPTIVE. NOT MISUSED. Don't know why I added this one.
Riddle - applies only to monika_tornment
Outfits - any topics mentioning clothing
Lewd - any topics mentioning with NSFW things
Concern - Monika expressing her concern (for player or in general)
Farewell/greeting - Self-explanatory
Apocalypse - all topics mentioning the concept of the world as we know it ending
Informative - topics containing interesting fatcs

**Comment by IsItBadToMakeAGitHubAccountJustForThis** [comment]
Somewhat like that.
A lot of tags were missed and it still requires a lot of work but I did what I could.

**Comment by therationalpi** [comment]
This is an excellent start. Just so we don't have too many categories (for UI reasons), I decided we should cut it down a bit. Here's the trimmed list that moniklover, Rune and I came up with:

Psychology, Society, School, Life, Media,Literature,Games Literature Club, Romance, Club Members,[Player],Monika, DDLC, Mod, Technology, Philosophy, Misc, Trivia

**Comment by IsItBadToMakeAGitHubAccountJustForThis** [comment]
I for some reason thought those work like tags on boorus and neded to simplify code changes.
Silly me.
Well, I guess I'll make a new pull then. Later.

**Pull request closed without merging**